src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.sarif

```diff
@@ -6,7 +6,7 @@
       "driver" : {
         "name" : "CodeQL",
         "organization" : "GitHub",
-        "semanticVersion" : "2.11.5",
+        "semanticVersion" : "2.11.6",

```

Comment:
Main WDDST is on CodeQL CLI version 2.11.5.

Reply:
Good catch.  It looks like I have 2.11.5 on my devbox for some reason.  I'll downgrade and re-run.